test/NuGet.Core.FuncTests/NuGet.Protocol.FuncTest/NuGet.Protocol.FuncTest.csproj

```diff
@@ -44,4 +45,4 @@
               ItemName="ReferencesCopiedInThisBuild" />
     </Copy>
   </Target>
-</Project>
+</Project>
```

Comment:
let's keep trailing new line. I thought our editorconfig actually specifies this